### PR TITLE
Adding small Enron Mobile dataset.

### PR DESCRIPTION
**Pull request by agutkin**
The dataset was developed by [Keith Vertanen](https://www.keithv.com/) and [Per Ola Kristensson](http://pokristensson.com/) and available here:

    https://www.keithv.com/software/enronmobile/

We don't store intermediate files, just the final data.

### CODE REVIEW

**Comment by sjjhsjjh** [comment]
Hi, Is there an license and copyright statement for the data? If not, then please can one be added. Cheers, Jim

**Comment by agutkin** [comment]
I wonder that as well, as I couldn't find any. Keith (@kdv123 ) is there any?

**Comment by kdv123** [comment]
The Enron mobile set is derivative of the original corpus:
https://www.cs.cmu.edu/~./enron/

At least according to this other derivative work, the original corpus is in the public domain:
http://wstyler.ucsd.edu/enronsent.html

What are we using the Enron mobile data for in the Dasher project? Normally I save this set for an evaluation test set rather than using it for training models. If we want a training corpus, I could generate something from Common Crawl or such.

**Comment by agutkin** [comment]
At the moment we're using twenty or so sentences for model bootstrapping a simple demo PPM model. Nothing fancy.

**Comment by sjjhsjjh** [comment]
Sorry if I missed it but is there a copyright and license in the code and data?
Having it here in the conversation isn't enough I think.

**Comment by sjjhsjjh** [comment]
Hi @agutkin , From here and elsewhere I got the impression that the PPM has switched to a different training set. In that case, is this PR still needed? If not, maybe you can close it. Cheers, Jim

**Comment by agutkin** [comment]
Hi @sjjhsjjh. I was thinking of implementing an ``auto-perplexity'' tool inside Dasher that automatically evaluates the current dynamic model on a fixed test set. This is where Enron data becomes useful. ***But***,  given that the `LICENSE` is a bit unclear at this stage, I agree that it makes sense to close this for now, and re-open when licensing is resolved.

**Comment by agutkin** [comment]
Closing.
